os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -332,8 +360,9 @@ int se_ameba_hal_get_hmac(hal_hmac_type mode, hal_data *input, uint32_t key_idx,
 	int ret;
 	inout_struc in_output;
 
-	if (key_idx >= KEY_STORAGE_INDEX_MAX) {	/* Index Out of Range */
-		return HAL_INVALID_SLOT_RANGE;
+	ret = se_ameba_index_chk(key_idx);
```

Comment:
key_idx out of range check missing here?

Reply:
All key index check will be done in se_ameba_index_chk().